src/tool/Option/OptionDlg.cpp

```diff
@@ -395,7 +339,33 @@ void COptionDlg::SettingLoad(CString szIniFile) {
     int iWindowCursor = GetPrivateProfileInt("Cursor", "WindowCursor", 1, szIniPath);
     m_Option.bWindowCursor = (iWindowCursor) ? true : false;
 }
+void COptionDlg::SettingServerLoad(CString szIniFile) {
+    if (szIniFile.GetLength() <= 0) {
+        return;
+    }
+
+    char szIniPath[_MAX_PATH] = "";
+    GetCurrentDirectory(_MAX_PATH, szIniPath);
+    lstrcat(szIniPath, "\\");
+    lstrcat(szIniPath, szIniFile);
+
+    m_ServerOption.Version = GetPrivateProfileInt("Version", "Files", 1298, szIniPath);
+}
+void COptionDlg::SettingServerSave(CString szIniFile) {
+    if (szIniFile.GetLength() <= 0) {
+        return;
+    }
+
+    char szIniPath[_MAX_PATH] = "";
+
+    GetCurrentDirectory(_MAX_PATH, szIniPath);
+    lstrcat(szIniPath, "\\");
+    lstrcat(szIniPath, szIniFile);
+
+    CString szBuff;
 
+    szBuff.Format("%d", m_ServerOption.Version); WritePrivateProfileString("Version", "Files", szBuff, szIniPath);
```

Comment:
Formatting is off here. Could you run `format.ps1` script and force push please?

Also let's use `std::format` here instead of this.

Reply:
I dont know how to use std::format i never do that, i trying but it should work. Maybe u can later do that in next PR. 


Thank you.